main/webapp/modules/core/scripts/project.js

```diff
@@ -184,6 +184,7 @@ Refine._showHideLeftPanel = function() {
 
 Refine.showLeftPanel = function() {
   $('div#body').removeClass("hide-left-panel");
+  if(ui.browsingEngine == undefined || ui.browsingEngine.resize == undefined) return;
```

Comment:
@wetneb : I know this is a cheap fix, if you have suggestions, I'm all ears…

Regards, A.